onprc_ehr/resources/queries/onprc_ehr/RandData_Birth.sql

```diff
@@ -0,0 +1,32 @@
+SELECT d.Id,
+       d.date,
+       d.date_type,
+       d.birth_condition,
+       d.room,
+       d.cge,
```

Comment:
This is causing a test failure. I assume it should be cage.

```suggestion
       d.cage,
```